octobot_trading/producers/simulator/recent_trade_updater_simulator.py

```diff
@@ -17,12 +17,18 @@
 
 from octobot_backtesting.data import DataBaseNotExists
 from octobot_channels.channels.channel import get_chan
+from octobot_trading.channels.exchange_channel import get_chan as get_exchange_chan
 
 from octobot_commons.channels_name import OctoBotBacktestingChannelsName
+from octobot_commons.enums import PriceIndexes
+from octobot_trading.constants import OHLCV_CHANNEL
+from octobot_trading.enums import ExchangeConstantsOrderColumns
 from octobot_trading.producers.recent_trade_updater import RecentTradeUpdater
 
 
 class RecentTradeUpdaterSimulator(RecentTradeUpdater):
+    SIMULATED_RECENT_TRADE_LIMIT = 2
```

Comment:
Can you explain why 2 recent trades to generate ?

Reply:
They are identical, it should change nothing to have 1 or 20 of them. 1 is not representative to me, at least 2 is better (can iterate) and I think more than 2 is unnecessary and takes time to process